tests/helpers/bitcoin.ts

```diff
@@ -2,3 +2,10 @@ export const ensureBitcoinBalance = async (client: any, blocks: number = 101) =>
   const balance = await client.getBalance()
   if (balance === 0) await client.generate(blocks)
 }
+
+export const ensureBitcoinBalanceAboveMin = async (client: any, minBalance: number = 0, blocks: number = 101) => {
+  const balance = await client.getBalance()
+  if (balance < minBalance) await client.generate(blocks)
+  else return
+  await ensureBitcoinBalanceAboveMin(client, minBalance, blocks)
```

Comment:
Why is this recursively calling itself?

Reply:
I wasn't sure how else to ensure the number of blocks mined would raise the balance above the min requested.

Reply:
ok, but If for some reason the generate fails silently, it will basically be in an infinite loop, blow the stack and crash the program. At a minimum, there would need to be some way to safely stop the recursion. This is a tricky problem in the case of bitcoin because the block rewards keeps getting halved over time.

Since this is a test helper and we start with a fresh blockchain, the block reward is 25 btc/block. run it once. Generating 101 blocks in the tests ensures there's way more btc than even the current min of 500. 

This is tricky to correctly test right now because the blockchain is currently shared between tests. Unless this test is always the first one, it's unlikely that the balance would ever be zero and trigger a low balance alert. This will probably change soon so we may want to comment out these tests and mark them as TODO.

Reply:
@warrenv How does it look now? Since we are sure that the `ensureBitcoinBalance` is going to generate 25*101 BTC and I have lowered the min to 1BTC is it OK to keep the: `should: 'return walletInfo property isBalanceLow as false',` test? I've added a commented out test before the `ensureBitcoinBalance` is called that will test for `should: 'return walletInfo property isBalanceLow as true',`

Reply:
Yes, that's looking much better!